### PR TITLE
V6 xpc dial

### PR DESCRIPTION
**Pull request by Clarence-F**
Added new XPC Dial Position page

### CODE REVIEW

File: sphinx-src/web-api/xpc-mode.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Expert Control Mode
+###################
+.. note:: Expert only
```
Comment (sjmh3) [inline sphinx-src/web-api/xpc-mode.rst:4]:
```suggestion

.. note:: Expert only
```

File: sphinx-src/web-api/xpc-mode.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+HTTP
+****
+
+GET
+===
+
+Gets the current mode dial position
```
Comment (sjmh3) [inline sphinx-src/web-api/xpc-mode.rst]:
```suggestion
Gets the current mode dial position.
```

File: sphinx-src/web-api/xpc-mode.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+     - Value Example
+   * - ``mode``
+     - string
+     - Enum; see options listed below
+     - ``"Run"``
+
+Expert Control Dial Enums
+*************************
+
+``Run``, ``Dmx1``, ``Dmx2``, ``Dali``, ``All``, ``Off``, ``PG1``, ``PG2``, ``PG3``, ``PG4``
```
Comment (sjmh3) [inline sphinx-src/web-api/xpc-mode.rst]: Why not a list?

**Review by sjmh3 — CHANGES_REQUESTED** [review]
You need to add this file to the index.

**Review by sjmh3 — CHANGES_REQUESTED** [review]
In fact, there's a whole chain of commits on this PR - this only needs 1 commit. We should review how you're creating these PRs.

**Comment by Clarence-F** [comment]
> In fact, there's a whole chain of commits on this PR - this only needs 1 commit. We should review how you're creating these PRs.

This was mainly because of how I branched out this file, I comitted everything all together originally, so when I branched off this one, it started with the first 4 commits, which I then had to unthread manually, which autocommitted every time. Just part of my learning process, I understand it better now, so won't happen again.

**Pull request closed without merging**